numpy/ma/extras.py

```diff
@@ -1657,7 +1659,7 @@ def flatnotmasked_contiguous(a):
         if not k:
             result.append(slice(i, i + n))
         i += n
-    return result or None
+    return tuple(result) or None
```

Comment:
Also update the docstring! (It states a list is returned)

Reply:
Fixed